build_tools/docker/cmake_swiftshader/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# An image for building IREE using CMake and testing IREE with SwiftShader
+# Vulkan implementation.
+
+# Build using:
+# docker build --tag gcr.io/iree-oss/cmake-swiftshader \
+#              build_tools/docker/cmake_swiftshader/
```

Comment:
```suggestion
# docker build --tag gcr.io/iree-oss/cmake-swiftshader \
#   build_tools/docker/cmake_swiftshader/
```
Is there a reason to have the extra large hanging indent here?

Reply:
Copy pasted. Removed.